lib/oli_web/live/collaboration_live/posts/list.ex

```diff
@@ -70,4 +70,15 @@ defmodule OliWeb.CollaborationLive.Posts.List do
     </div>
     """
   end
+
+  defp reply_margin(level) do
+    case level do
+      0 -> nil
+      1 -> "ml-[12px]"
+      2 -> "ml-[24px]"
+      3 -> "ml-[36px]"
+      4 -> "ml-[48px]"
+      _ -> "ml-[60px]"
+    end
+  end
```

Comment:
Personally, I don't like the padding after the second level. I think it's more aesthetic to have only one level of padding just for the replies of the root post. But that's just my opinion 😁